cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java

```diff
@@ -380,4 +393,26 @@ private static boolean isSpecialDirectory(Path dirName) {
         String dirNameStr = dirName.toString();
         return dirNameStr.startsWith(".") || dirName.toFile().isHidden() || ignoreDirs.contains(dirNameStr);
     }
+
+    /**
+     * Validates the org-name and package name.
+     *
+     * @param orgName The org-name
+     * @return True if valid org-name or package name, else false.
+     */
+    private static boolean validateOrg(String orgName) {
```

Comment:
Are we able to move this to a common location ?

Reply:
RepoUtils may be ?

Reply:
Moved the validateOrg and validatePkg methods to RepoUtils.java with af18af2